cgmanifest.json

```diff
@@ -6880,29 +6880,29 @@
       "component": {
         "type": "other",
         "other": {
-          "name": "influx-cli",
-          "version": "2.6.1",
-          "downloadUrl": "https://github.com/influxdata/influx-cli/archive/refs/tags/v2.6.1.tar.gz"
+          "name": "infinipath-psm",
```

Comment:
Thank you for fixing this.
I think the prior pr maybe was a manual edit of this file.

Reply:
No need to thank me - the ordering is happening automatically anyway.:) And I think it might have been more than just one manual edit. One day I'll turn the script into a PR check, git hook, or a PR bot. Something like that.:)